hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -484,8 +486,11 @@ private void printStats(PrintStream out) {
     out.println("Number of Volumes created: " + numberOfVolumesCreated);
     out.println("Number of Buckets created: " + numberOfBucketsCreated);
     out.println("Number of Keys added: " + numberOfKeysAdded);
-    out.println("Replication: " + replicationConfig.getReplication());
-    out.println("Replication type: " + replicationConfig.getReplicationType());
+    out.println("Replication: " + (replicationConfig != null ?
+        replicationConfig.getReplication() : OZONE_REPLICATION_DEFAULT));
+    out.println("Replication type: " + (replicationConfig != null ?
+        replicationConfig.getReplicationType() :
+        OZONE_REPLICATION_TYPE_DEFAULT));
```

Comment:
If `replicationConfig` is `null`, actual replication is set at server-side based on config.  It could be different from the defaults from `OzoneConfigKeys`.  I think it would be better to skip printing replication info in that case.

```suggestion
    if (replicationConfig != null) {
      out.println("Replication: " + replicationConfig.getReplication());
      out.println("Replication type: " + replicationConfig.getReplicationType());
    }
```

or even:

```suggestion
    if (replicationConfig != null) {
      out.println("Replication: " + replicationConfig);
    }
```